.vib/prestashop/goss/vars.yaml

```diff
@@ -0,0 +1,35 @@
+binaries:
+  - apachectl
+  - mariadb
+  - php
+  - render-template
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/prestashop
+      - /opt/bitnami/prestashop
+      - /opt/bitnami/prestashop/admin
+      - /opt/bitnami/prestashop/install
```

Comment:
I could not find these directories in the compilation or postunpack processes... I understand you check them because they are involved in the initialization process (which is okay 😁), but this step is out of the scope of the current approach. Am I right? Maybe I missed a reference somewhere.